AzureCalling/app/src/main/java/com/azure/samples/communication/calling/external/calling/CallingContext.java

```diff
@@ -468,8 +474,12 @@ private void unbindOnVideoStreamsUpdatedListener(final RemoteParticipant remoteP
     private void bindOnIsMutedChangedListener(final RemoteParticipant remoteParticipant) {
         final String username = remoteParticipant.getDisplayName();
         final String id = getId(remoteParticipant);
-        final PropertyChangedListener remoteIsMutedChangedListener = propertyChangedEvent ->
-                Log.d(LOG_TAG, String.format("Remote Participant %s addOnIsMutedChangedListener called", username));
+        final PropertyChangedListener remoteIsMutedChangedListener = propertyChangedEvent -> {
+            Log.d(LOG_TAG, String.format("Remote Participant %s addOnIsMutedChangedListener called", username));
+            remoteParticipantUpdate.postValue(new RemoteParticipantUpdate(remoteParticipant,
```

Comment:
We can do a check here if the participant that is muted/unmuted if actually being displayed or not. We only need to post the value if it's being displayed.

Reply:
Sure, done.